bots/example_bots/helpers/grid.py

```diff
@@ -0,0 +1,229 @@
+import math
+from _decimal import Decimal
+from typing import List, Optional, Tuple
+
+from rubi import OrderSide
+
+
+class DesiredOrder:
+    def __init__(
+        self,
+        price: Decimal,
+        size: Decimal,
+        side: OrderSide
+    ):
+        self.price = price
+        self.size = size
+        self.side = side
+
+
+class GridLevel:
+    def __init__(self, bid: DesiredOrder, ask: DesiredOrder):
+        self.bid = bid
+        self.ask = ask
+
+
+class Grid:
+
+    def __init__(
+        self,
+        # Inventory
+        starting_base_asset_amount: Decimal,
+        starting_quote_asset_amount: Decimal,
+        starting_base_asset_average_price: Optional[Decimal],
+        # Grid
+        fair_price: Decimal,
```

Comment:
discussed offline with @adamamsmith - the ability to update `fair_price`, with the necessary actions to happen when updated, will make this a very modular order management system that can be utilized in a variety ways!